site/docs/getting-started/installation.mdx

```diff
@@ -1,4 +1,17 @@
-# Configuring your cloud storage bucket for use with Flow
+---
```

Comment:
Does the change in this file's path also need a redirect?

Reply:
I have another PR to reverse that page back to its original form because of the Azure auth snippet. @jshearer, should we merge that first? Or would a redirect actually solve it too?